config/routes.rb

```diff
@@ -19,7 +19,8 @@
   resources :events do
     resources :bookings, only: [:create]
   end
-  resources :users
+  get '/session' => "sessions#create"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.